spec/support/macros/database_macros.rb

```diff
@@ -20,9 +20,9 @@ def run_migration(&block)
   end
 
   def setup_database(opts = {})
-    adapter = "#{opts[:adapter].capitalize}Adapter".constantize.new(database: opts[:database])
+    adapter = "#{opts[:adapter].classify}Adapter".constantize.new(database: opts[:database])
     adapter.establish_connection!
-    adapter.reset_database!
+    adapter.reset_database!(opts[:database])
```

Comment:
Oh I see we don't need to pass this here, we can use the attr_accessor.  I sent it because RuboCop complained about the instance variable, but if we an attr_accessor instead of the instance variable it should work fine without complaint.  Will fix.

Reply:
Fixed.